joern-cli/frontends/rubysrc2cpg/src/main/scala/io/joern/rubysrc2cpg/astcreation/RubyIntermediateAst.scala

```diff
@@ -121,7 +124,7 @@ object RubyIntermediateAst {
   final case class MethodDeclaration(methodName: String, parameters: List[RubyNode], body: RubyNode)(span: TextSpan)
       extends RubyNode(span)
       with ProcedureDeclaration
-      with AllowedTypeDeclarationChild
+      with AllowedTypeDeclarationChild {}
```

Comment:
Reason for this?

Reply:
Was originally putting offset directly in the MethodDecl which required the curlies, removed the curlies.